publications-2020.bib

```diff
@@ -1,5 +1,24 @@
 % Encoding: US-ASCII
 
+@article{Fehn2020,
+  title={High-order arbitrary {L}agrangian--{E}ulerian discontinuous {G}alerkin methods for the incompressible {N}avier--{S}tokes equations},
+  author={Fehn, Niklas and Heinz, Johannes and Wall, Wolfgang A and Kronbichler, Martin},
+  journal={arXiv preprint arXiv:2003.07166},
+  year={2020}
+}
+
+@article{Lorentzon2020,
+author = {Lorentzon, Johan and Revstedt, Johan},
+title = {A numerical study of partitioned fluid-structure interaction applied to a cantilever in incompressible turbulent flow},
+journal = {International Journal for Numerical Methods in Engineering},
+volume = {121},
+number = {5},
+pages = {806-827},
+keywords = {deal.II, FSI, LES, OpenFOAM, strongly coupled},
+doi = {10.1002/nme.6245},
```

Comment:
Can you please add an URL for consistency with the other entries?
```suggestion
doi = {10.1002/nme.6245},
url = {https://doi.org/10.1002/nme.6245},
```

Reply:
done